config/e2e/monitoring.yaml

```diff
@@ -82,6 +82,14 @@ spec:
       hosts: ['${monitoring_url}']
       username: ${monitoring_user}
       password: ${monitoring_pass}
+      # Since E2E cluster's Elasticsearch instance may lag behind the newest version of Beat, we need to allow this scenario.
+      allow_older_versions: true
+      # The maximum size to send in a single Elasticsearch bulk API index request. Needed if Elasticsearch is behind Cloud Ingress.
+      bulk_max_body_size: 10M
```

Comment:
I am confused:  have we not been running the monitor behind an ingress for a long time? Why is this only now a problem?

Reply:
Unsure what has recently changed, or changed with what I was ingesting, but I directly ran into:

```
client.go:244: ERR Failed to perform any bulk index operations: 413 Request Entity Too Large
```

during my testing of this during a local e2e test run.  This caused no ingestion at all into the ES cluster, and everything after failed.